content/docs/nav.yml

```diff
@@ -24,7 +24,7 @@
     - id: handling-events
       title: Handling Events
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: 條件渲染
```

Comment:
```suggestion
      title: 條件 rendering
```

Reply:
@bugtender, should we keep the original term `Render`?

```suggestion
      title: 條件 Render
```

Reply:
`Render` Yeah, sorry I forgot.😳